### ISSUE TITLE
Naming

### ISSUE THREAD

**Issue by djava** [purrleterian/Image-to-ASCII-art-converter#4]
I saw a bunch of naming things that break convention. 

IE using underscores instead of caps to indicate new words, you misspelled through as trough, and img_size is a misleading variable name

**Comment by purrleterian** [comment]
> indicate new words
What exactly do you mean by that? Should I just convert all my variables from snake_case to camelCase?

**Comment by djava** [comment]
Yeah, that's what I usually do although that's opinion based I guess. Also, constants should be all caps

**Comment by nogarcia** [comment]
I am not completely sure whether I like `camelCase`, `PascalCase`, or `snake_case` better. I am leaning more towards `snake_case`, but in case you decide to change it, I'd be happy to take up the task and open a PR.
